lib/ui-application/client.ioc-container.js

```diff
@@ -31,8 +32,9 @@ export function createClientIoCContainer(configData) {
     httpTransport: iocFactory(createAxiosTransport),
     pipelines: createServerPipeline,
     apiAdapter: iocFactory(createApiAdapter),
-
     app: iocClass(ClientUiApplication),
+    //manifestManager: iocClass(AssetsManifestManager),
```

Comment:
Why do you leave this comment `//manifestManager: iocClass(AssetsManifestManager),` ?